frontend/src/pages/about/About.tsx

```diff
@@ -0,0 +1,310 @@
+              contributors&apos; judgements on content quality. We hope to
+              contribute to making today&apos;s and tomorrow&apos;s large-scale
+              algorithms robustly beneficial for all of humanity. Find out more
+              with our white paper, our wiki, our github, and our Discord.{' '}
+              {/* TODO links */}
+            </Typography>
+          </ContentBox>
+        </Grid>
+      </Grid>
+
+      <Grid container className={classes.root}>
+        <Grid item xs={12} className={classes.container}>
+          <ContentBox>
+            <Typography variant="h1">Our Vision</Typography>
+            <Typography paragraph>...</Typography>
+          </ContentBox>
+        </Grid>
+      </Grid>
+
+      <Grid
+        container
+        className={classes.root}
+        style={{ background: '#1282B2', color: 'white' }}
+      >
+        <Grid item xs={12} className={classes.container}>
+          <ContentBox>
+            <Typography variant="h1">Who builds Tournesol?</Typography>
+            <Typography paragraph>
+              Tournesol seeks to be an extremely transparent project. All
+              contributions to the project are openly visible on our github,
+              description of important concepts and Tournesol’s vision can be
+              found on our wiki, and discussions that guide the development of
+              Tournesol happen openly on our Discord.
+            </Typography>
+          </ContentBox>
+        </Grid>
+      </Grid>
+
+      <Grid
+        container
+        className={classes.root}
+        style={{ background: '#FAF8F3' }}
+      >
+        <Grid item xs={12} className={classes.container}>
+          <ContentBox>
+            <img height="64px" src="/logos/Tournesol_Logo.png" />
+            <Typography variant="h4">Tournesol Association</Typography>
+            {/* TODO: Association logo */}
+
+            <Typography paragraph>
+              Tournesol is supported by the non-profit Tournesol association
+              based in Lausanne, Switzerland.
+            </Typography>
+          </ContentBox>
+        </Grid>
+
+        <Grid item xs={12} sm={4} lg={3} className={classes.container}>
+          <PeopleCard
+            name="Lê"
+            image="/people/Le.jpeg"
+            institution="IC School, EPFL"
+            role="President"
+            description="Dr. in Mathematics, AI Researcher and Communicator"
+          />
+        </Grid>
+
+        <Grid item xs={12} sm={4} lg={3} className={classes.container}>
+          <PeopleCard
+            name="Louis"
+            image="/people/Louis.jpeg"
+            institution="MSCI, Inc."
+            role="Treasurer"
+            description="Dr. in Computer Science, Software engineer"
+          />
+        </Grid>
+
+        <Grid item xs={12} sm={4} lg={3} className={classes.container}>
+          <PeopleCard
+            name="Aidan"
+            image="/people/Aidan.jpg"
```

Comment:
Typo in the filename:
```suggestion
            image="/people/aidan.jpg"
```

The image is also very large (> 1MB) and should be reduced/compressed.